gettingstarted/settings.py

```diff
@@ -50,6 +59,7 @@
     "django.contrib.auth.middleware.AuthenticationMiddleware",
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
+    "whitenoise.middleware.WhiteNoiseMiddleware",
```

Comment:
The order of `MIDDLEWARE` affects the order in which it runs, which can make a big difference.

The middleware is now being inserted at the end, whereas django-heroku inserted the WhiteNoise middleware at the beginning:
https://github.com/heroku/django-heroku/blob/fa2a5f7719625020a0dcf29654253f2982c26557/django_heroku/core.py#L105

...and in fact even at the beginning wasn't quite right. I would see the WhiteNoise docs for best practices for how to configure WhiteNoise - mentioned in the last review:
https://github.com/heroku/python-getting-started/pull/167#discussion_r925761263

After doing that, I would also add a comment here explaining what WhiteNoise does (given people new to Django or Heroku might not know, and it's something not in the template project produced by `django-admin startproject` etc):
```
# WhiteNoise handles serving static files in production.
```

Reply:
moved to under securitymiddleware like the docs